pkg/runtime/cli.go

```diff
@@ -160,6 +160,14 @@ func FromFlags() (*DaprRuntime, error) {
 		}
 	}
 
+	if applicationPort == daprHTTP {
+		return nil, errors.New("dapr api http port conflicts with app port")
```

Comment:
pls. use 'dapr-http-port' and ' and 'app-port' to align with user's input.



<br class="Apple-interchange-newline">

Reply:
The app port may be configured via Kubernetes annotations: `dapr.io/app-port`. Does this prompt affect the user's judgment?

Reply:
Agree, so should we mention both in the error message? @CrazyHZM

Reply:
Done